<launch>
    <node  name="raspicam" pkg="raspicam" type="raspicam_node" output="screen" >
        <param name="framerate" value="90" />
        <param name="quality" value="20" />
        <param name="width" value="640" />
        <param name="height" value="480" />
        <param name="tf_prefix" value="camera" />
    </node>
<!--
    <node name="republish" type="republish" pkg="image_transport" output="screen"
          args="compressed in:=/camera/image raw out:=/camera/image_raw" />

    <node pkg="image_proc" type="image_proc" name="image_proc" ns="camera" >
        <remap from="/camera/camera_info" to="/camera/camera_info" />
        <remap from="/camera/image/compressed" to="/camera/image_raw" />
    </node>
-->

<!--
    <node  name="robot_map_camera" pkg="usb_cam" type="usb_cam_node" output="log" >
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="1920" />
        <param name="image_height" value="1080" />
        <param name="pixel_format" value="mjpeg" />
        <param name="camera_frame_id" value="robot_map_camera" />
        <param name="io_method" value="mmap"/>
        <param name="framerate" value="30"/>

        <remap from="/usb_cam/camera_info" to="/robot_map_camera/camera_info" />
        <remap from="/usb_cam/image_raw" to="/robot_map_camera/image_raw" />

        <param name="camera_info_url"
               value="file://$(find raspberry_robot)/config/camera_info/camera_calibration_gopro_1920_1080.yaml"/>
    </node>
-->

</launch>